hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -247,9 +254,18 @@ private boolean checkBackupTimeout(Invocation invocation) {
             }
         }
 
-        private void log(int backupTimeouts, int invocationTimeouts) {
+        private void log(int invocationCount, int backupTimeouts, int invocationTimeouts) {
+            Level logLevel = null;
             if (backupTimeouts > 0 || invocationTimeouts > 0) {
-                logger.info("Handled " + invocationTimeouts + " invocation timeouts and " + backupTimeouts + " backupTimeouts");
+                logLevel = INFO;
+            } else if (logger.isFineEnabled()) {
```

Comment:
I'd guard it by `logLevel == null && logger.isFineEnabled`

as otherwise you are downgrading INFO to FINE -> confusing.